boards/arm/efr32bg_sltb010a/efr32bg_sltb010a.dts

```diff
@@ -43,6 +43,31 @@
 	};
 };
 
+&pinctrl {
+	compatible = "silabs,gecko-pinctrl";
```

Comment:
note: this is redundant, you're referencing an existing node (`&pinctrl`)

Reply:
done